besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -147,21 +148,38 @@ static class ExportTrieLog implements Runnable {
 
     @CommandLine.Option(
         names = "--trie-log-block-hash",
-        description = "The hash of the block you want to export the trie log of.",
+        description = "The hash of the block you want to import the trie log of",
```

Comment:
Think name and description need updating now it's a list.

Reply:
Updated the description. Think the name should still be the same following the same criteria of `rpc-http-api` for example

Reply:
Makes sense, thanks!